scripts/read-rush-json.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Finds and reads the rush.json file.
+ */
+function readRushJson() {
```

Comment:
>readRushJson [](start = 9, length = 12)

nice. I think the lint-imports script implements this function on its own.. it can probably import this

Reply:
Yeah I changed everything that reads rush.json to use this utility.

Reply:
BTW Rush provides a library `@microsoft/rush-lib` that includes an API for loading and querying the Rush configuration files.

---
In reply to: [236378415](https://github.com/OfficeDev/office-ui-fabric-react/pull/7186#discussion_r236378415) [](ancestors = 236378415)

Reply:
I'm hesitant to introduce a dependency on a large-ish library when we only need parsing. I did look a bit at rush-lib too (actually to try and see if it provided a programmatic way to *edit* rush.json and preserve comments), but without API docs it was hard to figure out what all it can do and how to use it.

Reply:
FWIW, TypeScript supports comments in tsconfig.json.